lib/scripts/controllers/options.js

```diff
@@ -56,8 +59,13 @@ export class OptionCtrl {
       console.log(data);
       this.patterns = JSON.parse(data.savedPatterns);
       this.key = data.yandexTranslatorApiKey;
+
       this.blacklistedWords = data.userBlacklistedWords.slice(1, -1).split('|');
+      this.blacklistedWords = this.blacklistedWords.filter(function(n){ return n != '' });
+
       this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
+      this.blacklistedWebsites = this.blacklistedWebsites.filter(function(n){ return n != '' });
```

Comment:
The blank word was being added because of the slicing and splitting not
because of blacklisting empty words.

Without merging my code, delete all the blacklisted words and refresh the
options page you'll see the blank entry.

On Saturday 11 June 2016, Bruno <notifications@github.com> wrote:

> In lib/scripts/controllers/options.js
> <https://github.com/OiWorld/MindTheWord/pull/131#discussion_r66701759>:
>
> >        this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
> > +      this.blacklistedWebsites = this.blacklistedWebsites.filter(function(n){ return n != '' });
>
> I agree with @ankit-m <https://github.com/ankit-m> on this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OiWorld/MindTheWord/pull/131/files/2313eec4b773f9ed65a3413141e4e4a8b38c2611#r66701759>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACVryqUwnC7FygcAFTbWQiJ2MTDs99nbks5qKlPLgaJpZM4IzexP>
> .
>


-- 
Regards,
Rohan Katyal

Please ignore the typos and brevity, email sent using a small form factor
device